usage/rest-client/src/test/java/org/apache/brooklyn/rest/client/BrooklynApiRestClientTest.java

```diff
@@ -98,11 +98,21 @@ public void testCatalogApiLocations() throws Exception {
         log.info("locations from catalog are: "+locations);
     }
 
+    public void testCatalogCreate()throws Exception {
+        final String bomYaml = "brooklyn.catalog:\n  id: simple-tomcat\n  version: 1.0\n  itemType: template\n" +
```

Comment:
Put a new line in after every `\n` to make this readable. Or alternatively put the blueprint in a file in src/test/resources/path/to/file.yaml and load it with `new ResourceUtils(this).getResourceAsString("classpath://path/to/file.yaml")`.